packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -14,24 +14,36 @@
 import yaml
 from robot.api.deco import library, keyword
 from robot.libraries.BuiltIn import BuiltIn
-
-from RPA.Email.ImapSmtp import ImapSmtp
-from RPA.FileSystem import FileSystem
 from RPA.core.helpers import import_by_name, required_env
 from RPA.core.logger import deprecation
 from RPA.core.notebook import notebook_print
-from .utils import (
+
+from RPA.Email.ImapSmtp import ImapSmtp
+from RPA.FileSystem import FileSystem
+from RPA.Robocorp.utils import (
     JSONType,
     Requests,
+    get_dot_value,
     is_json_equal,
     json_dumps,
     resolve_path,
+    set_dot_value,
     truncate,
     url_join,
 )
 
 
 UNDEFINED = object()  # Undefined default value
+ENCODING = "utf-8"
```

Comment:
```suggestion
ENCODING_UTF8 = "utf-8"
```

Reply:
Seems more explicit indeed, but what's the value brought here if I specify the `_<encoding-name>` in this constant? 

We usually work with one encoding only that's used everywhere else (so staying consistent), at least at `WorkItems` library level and this was introduced in order to DRY with the "utf-8" literal.